tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -54,3 +54,30 @@ Can Launch Python3 Smoke Test Notebook
   #Get the text of the last output cell
   ${output} =  Get Text  (//div[contains(@class,"jp-OutputArea-output")])[last()]
   Should Not Match  ${output}  ERROR*
+
+Verify A Default Image Is Provided And Starts Successfully
+    [Documentation]    Verify that, if a user doesn't explicitly select any jupyter image
+    ...    a default one is selected and it can be spawned successfully
+    [Tags]    Sanity
+    ...       ODS-469
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook
+    ${has_spawn_failed} =    Has Spawn Failed
+    Should Be Equal As Strings    ${has_spawn_failed}    False
+    Sleep    30s
+    Click Element    xpath=//div[@title='Python 3']
```

Comment:
why do you need this?

Reply:
For running `Spawned Image Check` , Without Opening any new tab , keyword is unable to add cell.

Reply:
After Spawning Notebook , Inorder to run  `Spawned Image Check`

Reply:
oh you need to select the kernel? does a dialog window pop out asking for selecting a kernel? you can try using `Maybe Select Kernel` kw

Reply:
No , I need to open Notebook

Reply:
there is a keyword to open a new notebook, try to use that

Reply:
Updated